src/test/scala/com/microsoft/hyperspace/index/IndexCacheTest.scala

```diff
@@ -166,18 +165,30 @@ class IndexCacheTest extends SparkFunSuite with SparkInvolvedSuite {
   }
 }
 
+class MockClock(startTime: Long = 1L) extends Clock {
```

Comment:
Can you add a small description of this class similar to `MockIndexCacheFactoryImpl` below?